packages/functional-tests/tests/oauth/webchannel.spec.ts

```diff
@@ -52,13 +52,9 @@ test.describe('severity-1 #smoke', () => {
     });
 
     test('signin', async ({
-      pages: { signin, relier, page },
+      syncBrowserPages: { signin, relier, page },
```

Comment:
Is this test using Sync? Sorry if that's a silly question, I don't see reference to it, so I'm double checking.

Reply:
The webchannels are for communicating with the browser/sync, yes.